nucypher/characters/base.py

```diff
@@ -161,10 +153,6 @@ def __init__(self,
         else:
             self._crypto_power = CryptoPower(power_ups=self._default_crypto_powerups)
 
-        # Fleet and Blockchain Connection (Everyone)
```

Comment:
:+1: - in the spirit of keeping defaults out of characters.  What in a sense "allowed" this change to occur now?

Reply:
Because we no longer load seednoes prior to init time; the entire fail fast logic is now init time.